interfaces/builtin/shared_memory.go

```diff
@@ -62,9 +69,18 @@ func validateSharedMemoryPath(path string) error {
 		return fmt.Errorf("shared-memory interface path has leading or trailing spaces: %q", path)
 	}
 
-	// TODO: allow "*" as a globbing character; figure out if more AARE should be allowed
-	if err := apparmor.ValidateNoAppArmorRegexp(path); err != nil {
-		return fmt.Errorf("shared-memory interface path is invalid: %v", err)
+	// allow specifically only a single "*" globbing character, but disallow all
```

Comment:
yes I can see this being misleading, single just means we allow a `*` but we don't allow a `**` since the latter in AARE will allow matching sub-dirs too

Reply:
I pushed an update to the comments